utils/api_info_v2.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+
+import re
+import os
+import argparse
+import requests
+import json
+
+DEV_HOST = 'http://127.0.0.1:5500'
+NYAA_HOST = 'https://nyaa.si'
+SUKEBEI_HOST = 'https://sukebei.nyaa.si'
+
+API_BASE = '/api'
+API_INFO = API_BASE + '/v2/info'
+
+ID_PATTERN = '^[1-9][0-9]*$'
+INFO_HASH_PATTERN = '^[0-9a-fA-F]{40}$'
+
+environment_epillog = '''You may also provide environment variables NYAA_API_HOST, NYAA_API_USERNAME and NYAA_API_PASSWORD for connection info.'''
+
+parser = argparse.ArgumentParser(description='Query torrent info on Nyaa.si', epilog=environment_epillog)
+
+conn_group = parser.add_argument_group('Connection options')
+
+conn_group.add_argument('-s', '--sukebei', default=False, action='store_true', help='Upload to sukebei.nyaa.si')
+
+conn_group.add_argument('-u', '--user', help='Username or email')
+conn_group.add_argument('-p', '--password', help='Password')
+conn_group.add_argument('--host', help='Select another api host (for debugging purposes)')
+
+resp_group = parser.add_argument_group('Response options')
+
+resp_group.add_argument('--raw', default=False, action='store_true', help='Print only raw response (JSON)')
+
+info_group = parser.add_argument_group('Info options')
+
+info_group.add_argument('-q', '--query', required=True, help='Torrent by id or hash Required.')
```

Comment:
Use a positional for this, because there's nothing else you can input - the main point is the ID/hash, just like in upload you have to have a torrent.